pages/linux/fdisk.md

```diff
@@ -10,3 +10,27 @@
 - Start the partition manipulator:
 
 `fdisk {{/dev/sdX}}`
+
+- Once partitioning a disk, create a partition:
+
+`n`
+
+- Once partitioning a disk, delete a specific partition:
+
+`d`
+
+- Once partitioning a disk, view the partition table:
+
+`p`
+
+- Once partitioning a disk, write the changes made:
+
+`w`
+
+- Once partitioning a disk, discard the changes made:
+
+`q`
+
+- Once partitioning a disk, open a help menu:
```

Comment:
I think this should be the first interactive mode command.

Reply:
Are you referring to the GPT table command or q or m

Reply:
I don't think it matters too much, but @navarroaxel is referring to the help `m` command.

Reply:
I prefer the help message to be at the bottom, and it makes sense anyway because the help command letter is hinted at the bottom always anyway